specification/app/resource-manager/Microsoft.App/preview/2023-04-01-preview/Jobs.json

```diff
@@ -935,6 +935,64 @@
       "format": "int32",
       "type": "integer"
     },
+    "PollingInterval": {
+      "description": "Interval to check each event source in seconds. Defaults to 30s",
+      "format": "int32",
+      "type": "integer"
+    },
+    "ScaleRule": {
```

Comment:
```suggestion
    "JobScaleRule": {
```

Reply:
This will not work because the requested json name for this is scaleRule.